igv_remote/_core.py

```diff
@@ -54,25 +48,27 @@ def _set_saveopts(self, img_dir, img_basename, img_init_id=0) :
         self._img_basename = img_basename
         self._img_id = img_init_id
     
-    def _set_viewopts(self, view_type, viewaspairs, sort):
+    def _set_viewopts(self, view_type,  sort):
         if view_type not in ['squished', 'collapsed', 'expanded']:
             raise ValueError("view_type must be one of [squished, collapsed, expanded]")
         self._view_type = view_type
-        self._viewaspairs = viewaspairs
         self._sort = sort
 
     def _connect(self, host="127.0.0.1", port=60151):
+        assert self.sock is None
```

Comment:
This also would be better as an exception; it's very possible for a careless user to try connecting twice, which would trigger this assertion. Remember, assertions are to catch bugs in code that an end user should never be able to trigger.